scripts/release.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
```

Comment:
What's the benefit of having a version file over importing the version from the `package.json`?

Reply:
When you import the package you get the entire package, that's not something we want.
Also, it puts all the compiled code under `src` folder (inside `dist`) but that's a matter of preference.
I can just clean the package after the build, your call.

Reply:
No it makes sense. Thanks for the explenation 👍